stable/artifactory-ha/templates/artifactory-primary-statefulset.yaml

```diff
@@ -367,10 +367,15 @@ spec:
         emptyDir:
           sizeLimit: {{ .Values.artifactory.persistence.size }}
       {{- else }}
+      {{- if .Values.artifactory.persistence.existingSharedClaim }}
```

Comment:
@danielezer It can't be used for `name: volume` as that is what each primary and member node use. This would end up with multiple member nodes all mounted to the same dir and conflicting

```
volumeMounts:
        - name: volume
          mountPath: "{{ .Values.artifactory.persistence.mountPath }}"
```

It would also mean the init-containers are copying their contents (like bootstrap) into the shared-drive at that same mountpath (usually /var/opt/jfrog/artifactory)